src/extension/messages/github.ts

```diff
@@ -0,0 +1,39 @@
+import { NotebookRendererMessaging } from 'vscode'
+
+import { ClientMessages } from '../../constants'
+import { ClientMessage } from '../../types'
+import { postClientMessage } from '../../utils/messaging'
+import { checkWorkflowRunStatus, deployWorkflow } from '../executors/github/workflows'
+
+export interface IGitHubMessaging {
+    messaging: NotebookRendererMessaging
+    message: ClientMessage<ClientMessages>
+}
+
+export default async function handleGitHubMessage({ messaging, message }: IGitHubMessaging): Promise<void> {
+    if (message.type === ClientMessages.githubWorkflowDispatch) {
```

Comment:
return early and safe indentations, e.g.:
```suggestion
    if (message.type !== ClientMessages.githubWorkflowDispatch) {
        return
    }
```

Reply:
Done